owncloudApp/build.gradle

```diff
@@ -3,48 +3,44 @@ apply plugin: 'com.android.application'
 def commitSHA1 = 'COMMIT_SHA1'
 def gitRemote = 'GIT_REMOTE'
 
-configurations.all {
-    resolutionStrategy {
-        force "com.android.support:support-annotations:${supportLibraryVersion}"
-    }
-}
-
 dependencies {
     /// dependencies for app building
     implementation project(':owncloud-android-library:owncloudComLibrary')
-    implementation "com.android.support:support-v4:${supportLibraryVersion}"
-    implementation "com.android.support:design:${supportLibraryVersion}"
+    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
+    implementation "androidx.legacy:legacy-support-v4:$androidX"
+    implementation "androidx.annotation:annotation:1.0.1"
```

Comment:
without this line, build "release" run into an issue. That's why build in "debug" was too less